src/modules/tests/test-data/cat-a-mod1/single-fault-competencies/single-fault-competencies.actions.ts

```diff
@@ -4,6 +4,8 @@ import { SingleFaultCompetencyNames } from '../../test-data.constants';
 
 export const SET_SINGLE_FAULT_COMPETENCY_OUTCOME = '[Single Fault Competency] Set Outcome';
 export const REMOVE_SINGLE_FAULT_COMPETENCY_OUTCOME = '[Single Fault Competency] Remove Outcome';
+export const REMOVE_SINGLE_SERIOUS_FAULT_COMPETENCY_OUTCOME = '[Single Fault Competency] Remove Outcome';
+export const REMOVE_SINGLE_DANGEROUS_FAULT_COMPETENCY_OUTCOME = '[Single Fault Competency] Remove Outcome';
```

Comment:
Would specifying if serious/dangerous fault help? Otherwise can use the same String since its exact match.

Reply:
Actually no, shouldn't be using the same string, but still maybe having more descriptive would help